facesign-service/Dockerfile

```
@@ -162,15 +196,18 @@ RUN sed -i 's#usageLogsDirectory: ""#usageLogsDirectory: "/mnt/encrypted/usage-l
+COPY ./facesign-service/package*.json ./
+COPY ./facesign-service/pnpm-workspace.yaml ./
+COPY ./facesign-service/pnpm-lock.yaml ./
+RUN pnpm install --prod --frozen-lockfile
 COPY ./facesign-service/ .
-RUN npm install
 
 # Download or copy all required secrets and setup environment
 WORKDIR /home/deploy
 
 # Setup PM2
 RUN npm install -g pm2
```

Hadolint (2.14.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)